website/source/guides/operations/upgrade/upgrade-specific.html.md

```diff
@@ -15,6 +15,84 @@ details provided for their upgrades as a result of new features or changed
 behavior. This page is used to document those details separately from the
 standard upgrade flow.
 
+## Nomad 0.9.0
+
+### Task Driver Plugins
+
+All task drivers have become plugins in Nomad 0.9.0. There are no user visible
+differences between 0.8 and 0.9 drivers except for LXC. There is a new method
+for client driver configuration options, but existing `client.options` settings
+are supported in 0.9. See [plugin configuration][plugin-stanza] for details.
+
+#### LXC
+
+LXC is now an external plugin and must be installed separately. See [the LXC
+driver's documentation][lxc] for details.
+
+### Structured Logging
+
+Nomad 0.9.0 switches to structured logging. Any log processing on the pre-0.9
+log output will need to be updated to match the structured output.
+
+Structured log lines have the format:
+
+```
+# <Timestamp> [<Level>] <Component>: <Message>: <KeyN>=<ValueN> ...
+
+2019-01-29T05:52:09.221Z [INFO ] client.plugin: starting plugin manager: plugin-type=device
+```
+
+Values containing whitespace will be quoted:
+
+```
+... starting plugin: task=redis args="[/opt/gopath/bin/nomad logmon]"
+```
+
+### HCL2 Transition
+
+Nomad 0.9.0 begins a transition to [HCL2][hcl2], the next version of the
+HashiCorp configuration language. While Nomad has begun integrating HCL2,
+users will need to continue to use HCL1 in Nomad 0.9.0 as the transition is
+incomplete.
+
+The only backward incompatible change is that `task.config` stanzas now only
+support dotted nested object notation in interpolation. Interpolating variables
```

Comment:
Maybe rewrite as "If you use variables with consecutive dots in them, you will need to change your job specification file to use the env map. See the following example:"?

Reply:
I like it but still want to mention it only affects task.config, so I changed it to:
> If you interpolate variables in your `task.config` containing consecutive dots
in their name, you will need to change your job specification to use the `env`
map. See the following example:

Also going to link to `/docs/job-specification/task.html#config`